index.html

```diff
@@ -215,6 +215,42 @@ <h5 class="text-white">Skills</h5>
               </ul>
             </div>
         </section>
+      <form action="https://formspree.io/f/xlezlkyw" method="post">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <input name="full-name" type="text" placeholder="Full name" maxlength="30" required>
+        <input name="email" type="email" placeholder="Email address" required>
+        <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] As required a user cannot submit the form if one or more of the fields is empty. So please add a `required` attribute as you did in the other fields.
- [ ] Please make the message text area field accepts up to 500 chars.

![image](https://user-images.githubusercontent.com/68796232/186274108-691eaaa9-4848-4e84-a395-96375ac85929.png)